src/quest/__init__.py

```diff
@@ -34,3 +33,31 @@ def create_history(wid: str) -> History:
         return PersistentHistory(wid, LocalFileSystemBlobStorage(save_folder / namespace / wid))
 
     return WorkflowManager(namespace, storage, create_history, factory)
+
+def create_sql_manager(
+        db_url: str,
+        namespace: str,
+        factory: WorkflowFactory
+) -> WorkflowManager:
+
+    database = SQLDatabase(db_url)
+
+    storage = SqlBlobStorage(namespace, database.get_engine())
+
+    def create_history(wid: str) -> History:
+        return PersistentHistory(wid, SqlBlobStorage(wid, database.get_engine()))
+
+    return WorkflowManager(namespace, storage, create_history, factory)
+
+def create_dynamodb_manager(
+        namespace: str,
+        factory: WorkflowFactory,
+) -> WorkflowManager:
+    dynamodb = DynamoDB()
+
+    storage = DynamoDBBlobStorage(namespace, dynamodb.get_table())
+
+    def create_history(wid: str) -> History:
+        return PersistentHistory(wid, DynamoDBBlobStorage(wid, dynamodb.get_table()))
+
+    return WorkflowManager(namespace, storage, create_history, factory)
```

Comment:
Look into how to make these available only when the add-on is installed.

Reply:
Working on it, seems like there are a few ways to do it, but dependencies groups in poetry seems like the most promising.